spec/features/suggested_salary_searches_spec.rb

```diff
@@ -0,0 +1,295 @@
+          stub_suggested_without_salary_filter(radius: 50, number_of_results: 4)
+          stub_suggested_with_salary_filter(radius: 50, number_of_results: 10)
+        end
+
+        it "displays a suggested search for the non-salaried courses with the larger radius first followed with salaried in a larger radius" do
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 20))
+
+          expect(results_page.suggested_search_heading.text).to eq("Suggested searches")
+          expect(results_page.suggested_search_description.text).to eq("You can find:")
+          expect(results_page.suggested_search_links.first.text).to eq("4 courses including non salaried within 50 miles")
+          expect(results_page.suggested_search_links.last.text).to eq("10 courses within 50 miles")
+        end
+      end
+    end
+
+    context "And the initial search was filtered to a 50 mile radius" do
+      before do
+        stub_results_with_salary_filter(radius: 50, number_of_results: 0)
+      end
+
+      context "with courses available that are non-salaried with the same radius and salaried in a larger radius" do
+        before do
+          stub_suggested_without_salary_filter(radius: 50, number_of_results: 4)
+          stub_suggested_across_england_with_salary_filter(number_of_results: 10)
+        end
+
+        it "displays a suggested search for the non-salaried courses with the same radius first followed with salaried in a larger radius" do
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 50))
+
+          expect(results_page.suggested_search_heading.text).to eq("Suggested searches")
+          expect(results_page.suggested_search_description.text).to eq("You can find:")
+          expect(results_page.suggested_search_links.first.text).to eq("4 courses including non salaried within 50 miles")
+          expect(results_page.suggested_search_links.last.text).to eq("10 courses across England")
+        end
+      end
+
+      context "with courses available that are non-salaried with a larger radius and salaried in a larger radius" do
+        before do
+          stub_suggested_without_salary_filter(radius: 50, number_of_results: 0)
+          stub_suggested_across_england_without_salary_filter(number_of_results: 4)
+          stub_suggested_across_england_with_salary_filter(number_of_results: 10)
+        end
+
+        it "displays a suggested search for the non-salaried courses with the larger radius first followed with salaried in a larger radius" do
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 50))
+
+          expect(results_page.suggested_search_heading.text).to eq("Suggested searches")
+          expect(results_page.suggested_search_description.text).to eq("You can find:")
+          expect(results_page.suggested_search_links.first.text).to eq("4 courses including non salaried across England")
```

Comment:
![image](https://user-images.githubusercontent.com/470137/77565453-2b71b280-6ebc-11ea-9d4f-ce9307837ada.png)


Wording  `including non salaried` ...

Reply:
Thanks! I was going to grab the wording once I'd fixed up the other comments :)